README.md

```diff
@@ -10,11 +10,13 @@ Before converting:
 3. Install `solusio` module and configure its API connection. [more](https://hostbill.atlassian.net/wiki/spaces/DOCS/pages/1723334657/SolusVM+2)
 4. Create new order page for SolusVM 2 products with properly order type [admin -> Settings -> Products & Services]. [more](https://hostbill.atlassian.net/wiki/spaces/DOCS/pages/2004287489/SolusIO+-+VPS+Package+-+pre-paid+billing)
 
-Steps of converting:
+Install the converter:
 1. Download zip-archive with converter: `wget https://github.com/solusio/hostbill-converter/archive/refs/heads/main.zip`
 2. Extract downloaded archive to Hostbill modules directory: `unzip main.zip -d [path to your application]/includes/modules/Hosting`.
 3. Go to converter directory `cd [path to your application]/includes/modules/Hosting/hostbill-converter`.
-4. Run `php index.php prepare` to load necessary data from SolusVM 2.
+
+Full converting:
```

Comment:
converting -> conversion

Remove sentence "This is one-way conversion - it will not be possible to restore converted SolusVM 2 product back to SolusVM 1." from the beginning of the README and add it here.